libs/shared-atomic-design-components/src/lib/atoms/Checkbox.tsx

```diff
@@ -5,16 +5,18 @@ import './Checkbox.scss'
 
 interface Props {
   name: string
-  value: string
+  value?: string
   checked: boolean
-  children: React.ReactNode
-  disabled: boolean
+  children?: React.ReactNode
+  disabled?: boolean
+  handleChange?: (...args: any[]) => void
+  handleBlur?: (...args: any[]) => void
+  isSubmitting?: boolean
   className?: string
   customOnChange?: Function
```

Comment:
I'd also go ahead and remove `customOnChange` in favor of `handleChange`. Of course by finding the instances of this component within the project and replacing `customOnChange` keyword with `handleChange`. What do you think?